src/assets/contributors-map.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "first_name": "yun",
+    "last_name": "ji",
+    "github_id": "actuallyyun",
+    "home_coordinates": "41.26613040056221, 0.905152180850496",
```

Comment:
FYI for privacy reason you don't have to share your exact home coordinates, a city landmark, a nearby crag is good enough.

Reply:
Good point! I will put a note in the instructions.